src/infrastructure/webhook/webhook-service.ts

```diff
@@ -29,9 +29,10 @@ export class WebhookService {
 
 		// Ensure team admin OAuth2 credentials are still valid
 		try {
-			await figmaService.getValidCredentialsOrThrow(
-				figmaTeam.figmaAdminAtlassianUserId,
-			);
+			await figmaService.getValidCredentialsOrThrow({
```

Comment:
/nit We construct a `ConnectUserInfo` from `FigmaTeam` in a few places, would it be worth having a helper to do this? Could be a method on `FigmaTeam` or `ConnectUserInfo`? Though they would need to be converted to classes, so maybe it's not worth the effort.

Reply:
That's a good point -- I agree, Justin! Let me address this in a separate PR.